### PR TITLE
feat(security-code): add support for security code for stored card me…

### PR DESCRIPTION
**Pull request by douglaseggleton**
# Description

Adds the option `requireSecurityCode` for embed to prompt for security codes on stored cards..

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own changes
- [x] I have run `yarn lint` to make sure my changes pass all tests
- [x] I have run `yarn test` to make sure my changes pass all linters
- [x] I have pulled the latest changes from the upstream `main` branch
- [x] I have tested both the react and the CDN versions on local and integration environments
- [x] I have added the necessary labels to this PR in case a new release needs to be published after merging into `main` (e.g. `release` and `patch`)

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

File: packages/embed/webpack.dev.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ module.exports = {
       title: 'Gr4vy - Embed',
     }),
     new DefinePlugin({
-      PACKAGE_VERSION: 'dev',
+      PACKAGE_VERSION: JSON.stringify('dev'),
```
Comment (cbetta) [inline packages/embed/webpack.dev.js:54]: @douglaseggleton what's this for?

**Review by cbetta — APPROVED** [review]

File: packages/embed/webpack.dev.js (hunk 1 above)
Comment (douglaseggleton) [reply packages/embed/webpack.dev.js:54]: We introduced package version but it doesn't work with `dev` as its not actually defined, it should be a string, e.g. `"dev"`

Comment (cbetta) [reply packages/embed/webpack.dev.js:54]: Isn't `'dev'` already a string?

Comment (douglaseggleton) [reply packages/embed/webpack.dev.js:54]: `PACKAGE_VERSION: '"dev"'`

**Comment by douglaseggleton** [comment]
@cbetta This is the error you get locally when it is set to just `'dev'`

Screenshot 2022-08-31 at 20 11 17

File: packages/embed/webpack.dev.js (hunk 1 above)
Comment (cbetta) [reply packages/embed/webpack.dev.js:54]: So confused. Don't let this top merging though.

Comment (douglaseggleton) [reply packages/embed/webpack.dev.js:54]: Take a look at https://webpack.js.org/plugins/define-plugin/#usage - Will probably explain it better than me! :D

**Review by thomas-gr4vy — APPROVED** [review]
I was unable to test this locally but looks good from the diff.

**Comment by gr4vy-code** [comment]
:rocket: PR was released in [`v2.12.0`](https://github.com/gr4vy/gr4vy-embed/releases/tag/v2.12.0) :rocket:

**Pull request merged**